src/main/resources/messages.properties

```
@@ -1196,17 +1196,19 @@ provider-response-submit-complete.notice-no=We will notify the CCR&R of your res
 provider-response-submit-complete.button-no=Return to home
 
 #info
```

Even though it's duplicative, I think this will enable content team to change copy here without affecting other areas.
```suggestion
#info
provider-response-info.title=Tell us about yourself
provider-response-info.header=Tell us about yourself
```